manta-pay/src/crypto/ecc/arkworks.rs

```diff
@@ -23,9 +23,11 @@ use ark_r1cs_std::ToBitsGadget;
 use ark_relations::ns;
 use ark_serialize::{CanonicalDeserialize, CanonicalSerialize, SerializationError};
 use core::marker::PhantomData;
+use derivative::Derivative;
```

Comment:
I'm using the convention to inline this macro at the call site to reduce the number of imports, since this macro is used a lot, and importing this does not reduce the complexity at the call site:

```rust
#[derive(derivative::Derivative)]
```

Reply:
done